packages/admin-e2e-tests/package.json

```diff
@@ -0,0 +1,50 @@
+{
+ "name": "@woocommerce/admin-e2e-tests",
+ "version": "0.1.0",
+ "author": "Automattic",
+ "description": "E2E tests for the new WooCommerce interface.",
+ "homepage": "https://github.com/woocommerce/woocommerce-admin/tree/main/packages/admin-e2e-tests/README.md",
+ "repository": {
+  "type": "git",
+  "url": "https://github.com/woocommerce/woocommerce-admin.git"
+ },
+ "keywords": [
+  "woocommerce",
+  "e2e"
+ ],
+ "license": "GPL-3.0+",
+ "main": "build/index.js",
+ "types": "build/index.d.ts",
+ "files": [
+  "/build/",
+  "!*.ts.map",
+  "!*.tsbuildinfo"
+ ],
+ "sideEffects": false,
+ "scripts": {
```

Comment:
Are these scripts needed? I don't think the other packages have their own scripts.

Reply:
> Are these scripts needed?

@adrianduffell I removed these and it looks fine. This is ready for another review. Note that I updated some of the package manifests to eliminate linting warnings. The linting log was over 1000 lines and the missing `@wordpress/*` package dependencies were contributing a considerable amount.

Reply:
@rrennick Thanks! I'll review this later today.

Reply:
Thanks for fixing the linting log 👍